### DISCUSSION TITLE
If you search for a word inside your notes, it should jump to the spot where the word is rather than just tell you the word is somewhere on that page

### DISCUSSION THREAD

**Discussion by ars4l4n** [standardnotes/forum#2721] [Suggestions]
That becomes a problem when you work with really long notes which is usually what I do for my creative work.
Feels more intuitive to me when it works this way. 

I'd love to do the switch from OneNote to this one for a better performance, cleaner UI, more appealing dark theme options (and hopefully less bugs) but this is definitely a dealbreaker to me.

**Comment by JaspalSuri** [comment]
Hi @ars4l4n, thanks for the feedback! We plan to improve the search experience in future updates to the app, hopefully around the time when version 4.0 or our desktop and mobile apps are released.

**Comment by hayguen** [comment]
Hi,
i have same problem: long notes, i need to search inside.
This looks to be the only missing simple function, which is a blocker for me, switching to standard notes.

Currently testing the free/standard edition.

From the FAQ and the web demo (https://standardnotes.org/demo), it looks only the 'Minimal Markdown Editor' has the search inside a note function.
* will that also be available with the free 'Plain Editor' version?
* more important: will there be a simple 'Find Next' / 'Find Prev' function jumping to that position by simple keyboard shortcut, e.g. F3 or Ctrl+G ? If yes, in which of Free/Extended versions might it be available?
* for when is the version 4.0 coarsely planned?

Thanks in advance
